packages/ckeditor5-table/src/tablecolumnresize/utils.js

```diff
@@ -280,11 +266,11 @@ function calculateMissingColumnWidths( columnWidthsAttribute ) {
 		.map( columnWidth => toPrecision( columnWidth ) );
 }
 
-// Inserts column resizer element into a view cell.
+// Inserts column resizer element into a view cell if it is missing.
 //
 // @param {module:engine/view/downcastwriter~DowncastWriter} viewWriter View writer instance.
-// @param {module:engine/view/element~Element} viewCell View cell.
-export function insertColumnResizerElement( viewWriter, viewCell ) {
+// @param {module:engine/view/element~Element} viewCell View cell where resizer should be put.
+export function ensureColumnResizerElement( viewWriter, viewCell ) {
```

Comment:
Correct me if I'm wrong, but exported functions should have docs that start with `*` and not with `//` :thinking: